kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -213,6 +213,8 @@
 
   private Boolean autoConfigure = Boolean.FALSE;
 
+  private String file;
```

Comment:
Why don't we store it as a `File`?
e.g. `private File loadedKubeConfig;`

Reply:
Nice idea. I considered using `Path`, which is the newer more versatile API but kept `File` since it fits more nicely into the existing code (less changes).